testing/jormungandr-integration-tests/Cargo.toml

```diff
@@ -5,10 +5,12 @@ authors = [ "dev@iohk.io" ]
 edition = "2018"
 
 [dependencies]
-grpc            = "0.8"
-grpc-protobuf   = "0.8"
-protobuf        = "2.14"
-futures         = "0.3"
+tonic = "0.2"
+prost = "0.6"
+tokio = { version = "0.2", features = ["macros"] }
+futures      = "0.3.5"
+futures-core = "0.3.5"
+futures-util = "0.3.5"
```

Comment:
The definitions in these two are available through `futures`, no need to manage them separately here.

Reply:
i think i don't really understand, if i remove those to lines i'm getting compilation issue:

```
error[E0432]: unresolved import `futures_util`
  --> testing\jormungandr-integration-tests\src\mock\client.rs:14:5
   |
14 | use futures_util::stream;
   |     ^^^^^^^^^^^^ use of undeclared type or module `futures_util`

error[E0432]: unresolved import `futures_core`
  --> testing\jormungandr-integration-tests\src\mock\server.rs:29:5
   |
29 | use futures_core::Stream;
   |     ^^^^^^^^^^^^ use of undeclared type or module `futures_core`

```

Reply:
Right, just replace these with:

```rust
use futures::stream;
```
```rust
use futures::Stream;
```

And you are good to go. Using futures subcrates directly is for libraries that want to minimize their dependency graphs.